components/Layout/SideNavbar/SideNavbar.tsx

```diff
@@ -21,16 +22,25 @@ const SideNavbar: React.FunctionComponent = () => {
           onClick={() => {
```

Comment:
Could you remove the onClick function for the Sunday Friends logo? I think we agreed on that the logo should not link you anywhere